### PR TITLE
Fix building from source

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

**Review by krishung5 — APPROVED** [review]

**Review by CoderHam — DISMISSED** [review]

**Review by tanmayv25 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -148,9 +148,10 @@ the build. If the CMake variables below are not specified, "main" branch
 of those repositories will be used. \<GIT\_BRANCH\_NAME\> should be the same
 as the Python backend repository branch that you are trying to compile.
 
-* triton-inference-server/backend: -DTRITON\_BACKEND\_REPO\_TAG=\<GIT\_BRANCH\_NAME\>
-* triton-inference-server/common: -DTRITON\_COMMON\_REPO\_TAG=\<GIT\_BRANCH\_NAME\>
-* triton-inference-server/common: -DTRITON\_COMMON\_REPO\_TAG=\<GIT\_BRANCH\_NAME\>
+* triton-inference-server/backend: -DTRITON_BACKEND_REPO_TAG=\<GIT\_BRANCH\_NAME\>
```
Comment (CoderHam) [inline README.md:151]: `-DTRITON_BACKEND_REPO_TAG=<GIT_BRANCH_NAME>`  for better readability?

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Tabrizian) [reply README.md:151]: I don't understand the difference.

Comment (CoderHam) [reply README.md:151]: I should have been clear. Can you wrap it in "`"?

Comment (Tabrizian) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -148,9 +148,10 @@ the build. If the CMake variables below are not specified, "main" branch
 of those repositories will be used. \<GIT\_BRANCH\_NAME\> should be the same
 as the Python backend repository branch that you are trying to compile.
 
-* triton-inference-server/backend: -DTRITON\_BACKEND\_REPO\_TAG=\<GIT\_BRANCH\_NAME\>
-* triton-inference-server/common: -DTRITON\_COMMON\_REPO\_TAG=\<GIT\_BRANCH\_NAME\>
-* triton-inference-server/common: -DTRITON\_COMMON\_REPO\_TAG=\<GIT\_BRANCH\_NAME\>
+* triton-inference-server/backend: `-DTRITON\_BACKEND\_REPO\_TAG=\<GIT\_BRANCH\_NAME\>`
```
Comment (CoderHam) [inline README.md:151]: Since you are using "`" you can get rid of the \ before _

**Review by CoderHam — CHANGES_REQUESTED** [review]

**Pull request merged**